packages/@okta/vuepress-site/docs/guides/customize-tokens-returned-from-okta/add-groups-claim-dynamic/index.md

```diff
@@ -11,6 +11,8 @@ You can use this function anywhere to get a list of Groups of which the current
 | pattern            | Search term                                   | FALSE          | `"Eastern-Region"`, `"Eastern"`, `"-Region"`|
 | limit              | Maximum number of Groups returned. Must be a valid EL expression and evaluate to a value between 1 to 100. | FALSE | `1`, `50`, `100`|
 
+> **Important:** When you use `Groups.startWith`, `Groups.endsWith`, or `Groups.contains`, the `pattern` argument is matched and populated on the `name` attribute rather than the group's email. If you are targeting groups that may have duplicate group names (such as Google Groups), use the `getFilteredGroups` Group function instead. See the **Parameter Examples** section of [Use group functions for static group whitelists](/docs/guides/customize-tokens-returned-from-okta/static-whitelist/#use-group-functions-for-static-group-whitelists) for more information on the parameters used in this Group function.
```

Comment:
I don't understand the part about "the group's email". I know groups have names, but what does it mean for a group to have an email?

Reply:
@motikagan-okta - apparently in google groups, the group is associated with not just a name but an email address as well. Perhaps I need to make that more clear - that we're talking specifically about Google Groups? That's the use case for this change - it was G Suite-specific.

Reply:
Okay, yes, if this is a Google-specific thing, it would be good to say that. Is it that Google as IdP offers two ways of referring to  a group: a group name or a group email address?

Reply:
Is this relevant, in the Google docs: https://support.google.com/a/answer/33343?hl=en&ref_topic=25840
They talk about "Group Name" and "Group Email".